packages/instantsearch.js/scripts/rollup/rollup.config.js

```diff
@@ -25,6 +25,7 @@ const plugins = [
     rootMode: 'upward',
     exclude: /node_modules/,
     extensions: ['.js', '.ts', '.tsx'],
+    sourceType: 'unambiguous',
```

Comment:
I also wonder if the helper can be considered external? maybe that helps (although I'm not sure)